content/en/guides/diversity-inclusion.md

```diff
@@ -0,0 +1,173 @@
+
+**Listen.** Accepting begins with listening and so does issue resolution. Sometimes misunderstandings happen, especially where language and cultural differences exist. It’s important to sort these out before anyone reacts to anything. Active listening also means listen to everyone with an open mind. Don’t discount ideas or complaints – or accept excuses -- out of hand. Look into issues and search for solutions. Then look again for ways to ensure the environment isn’t fostering such behaviors elsewhere in the community.
+
+**Be kind.** Even old timers in the community will feel more relaxed and respected when kindness prevails. Newcomers will feel even more so. However, inclusiveness is more than just good manners. People can still be mistreated without voices being raised or threats made directly. You can combat this with sustained and expressed emphasis on kindness and professional courtesies. 
+
+**Adopt a code of conduct and enforce it**. In reality, bad behavior does happen in open source communities, whether or not it is intentional. Situations can get heated and communications may fall short of professional standards. Small acts of misconduct can easily escalate in an online community and harm project participants or [the project itself](http://writing.jan.io/2015/11/20/sustainable-open-source.html). A Code of Conduct (CoC) is a social contract that encourages good behavior and discourages bad behavior in a community. A CoC helps address these issues and how they are to be resolved. It isn’t always easy to get all members of a community to accept a CoC, but it’s almost impossible to take steps to make and maintain a healthy, diverse and inclusive community without a CoC that can be, and is, enforced. Here is [a good template](https://www.contributor-covenant.org/) to use, along with a great [blog](https://medium.com/mozilla-open-innovation/how-were-making-code-of-conduct-enforcement-real-and-scaling-it-3e382cf94415) describing an example of an enforcement workflow.
+
+A CoC addresses these types of issues:
+
+* when people disagree about a very contentious issue
+
+* when someone checks in code that breaks an important feature
+
+* when people join a project midstream and lack context about an old issue
+
+* when people are unclear in their communication style
+
+* when language and culture cause people to misunderstand
+
+* when someone feels the need to leave the community
+
+* when insults, threatening words or unwanted sexual attention occurs
+
+"Some people think the Code of Conduct is there to weed out people or to police online behaviors," says [Gil Yehuda](https://www.linkedin.com/in/gilyehuda/), senior director of Open Source at Verizon Media. “I think Codes of Conduct are more method for reminding all people to act with respect, even when things are stressing the community. A project leader can point to the code before something unravels into a fight, and that’s better than waiting for a complaint to be issued.”
```

Comment:
Gil was already introduced earlier. Linking here too is probably fine, but it probably does need to have the full role introduction at this point.